.travis.yml

```
@@ -0,0 +1,12 @@
+language: java
+dist: bionic
+sudo: false
+jdk:
+  - openjdk11
+env:
+  - GH_URL=https://raw.githubusercontent.com FILE_TO_VALIDATE=spec/search-api.yaml URL_TO_VALIDATE=$GH_URL/${TRAVIS_PULL_REQUEST_SLUG:-$TRAVIS_REPO_SLUG}/${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}/$FILE_TO_VALIDATE
+before_install:
+  - git clone --branch=v1.0.1 https://github.com/mcupak/oas-validator.git
+  - ./oas-validator/setup-validator.sh
+script:
+  - ./oas-validator/validate.sh "$URL_TO_VALIDATE"
```

RE chat on twitter, if you wanted spectral you could do:

```suggestion
  - npx @stoplight/spectral lint "$URL_TO_VALIDATE"
```

Then just delete the `before_install` and you're all set.